make/assembly2db.mk

```diff
@@ -0,0 +1,59 @@
+# Copyright (C) 2012-2022 SUSE Software Solutions Germany GmbH
+#
+# Author:
+# Frank Sundermeyer <fsundermeyer at opensuse dot org>
+#
+# Convert DocBook assemblies to a DocBook bigfile ("realized XML")
+#
+# Please submit feedback or patches to
+# <fsundermeyer at opensuse dot org>
+#
+
+include $(DAPSROOT)/make/common_variables.mk
+
+# Stylesheets and Settings
+
+STYLEASSEMBLY  := $(firstword $(wildcard $(addsuffix \
+	        /assembly/assemble.xsl, $(STYLE_ROOTDIRS))))
```

Comment:
`/usr/share/xml/docbook/stylesheet/nwalsh5/current/assembly/assemble.xml`

from the docbook-stylesheets package. In case you use your own stylesheets specified with STYLEROOT and in case this directoty contains assembly/assemble.xsl it will be used.

Reply:
I see, thanks!

What would be the reason for using a custom assemble.xml file? Or, in other words, wouldn't it be better to always use the upstream stylesheet here?

Reply:
The upstream stylesheets _are_ used unless you provide a custom,, assembly file.

As for  why using a custom file--have a look at assembl.xsl. It is work in progress, some features are not yet supported (e.g. <relationship>) and there are bugs (e.g. not transferring xml:lang from the <assembly> tag).